Docs/source/usage/parameters.rst

```diff
@@ -454,6 +454,14 @@ Particle initialization
       The ``external_file`` option is currently implemented for 2D, 3D and RZ geometries, with record components in the cartesian coordinates ``(x,y,z)`` for 3D and RZ, and ``(x,z)`` for 2D.
       For more information on the `openPMD format <https://github.com/openPMD>`__ and how to build WarpX with it, please visit :ref:`the install section <install-developers>`.
 
+    * ``NFluxPerCell``: Continuously inject a flux of macroparticles from a planar surface.
```

Comment:
Hi @dpgrote, just for my understanding: how do we handle the fields? Is this usually combined with a metallic boundary condition on that surface?

Reply:
This PR makes no assumptions about the fields. Presumably, the user would be using metallic boundary, but this would depend on the application. Note that the emission surface does not need to lie on one of the domain boundaries, but can be anywhere.